types/react-payment-inputs/index.d.ts

```diff
@@ -0,0 +1,316 @@
+// Type definitions for react-payment-inputs 1.1
+// Project: https://github.com/medipass/react-payment-inputs
+// Definitions by: Timothy Odei Yirenkyi <https://github.com/tyirenkyi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import React = require("react");
```

Comment:
```suggestion
import * as React from 'react';
```
this import was ok, but just use that one to follow convention here (like in eg. react-router, etc) to use this 'working everywhere' pattern